specification/api-tracing.md

```diff
@@ -386,6 +387,18 @@ Attributes SHOULD preserve the order in which they're set. Setting an attribute
 with the same key as an existing attribute SHOULD overwrite the existing
 attribute's value.
 
+Attribute values expressing a numerical value of zero or an empty string are
+considered meaningful and MUST be stored and passed to the exporter.
+Attribute values of `null` are considered to be not set and discarded as if
+`SetAttribute` was never called.
+If overwriting values is allowed, this results in clearing the previous value
```

Comment:
What's the general recommendation on overwriting?
From a DT perspective, this is completely unsupported at the moment. Should we try and discourage that in the spec?

Reply:
Currently it's a SHOULD: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-tracing.md#L385-L387

Reply:
It's probably hard to argue why this should be forbidden in general 🤔

Reply:
Actually I'm not sure if this was introduced to solve a legit use case or if it was just added as nice to have without considering YAGNI.